src/js/components/homepage/features/CovidFeature.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * CovidFeature.jsx
+ * Created By Jonathan Hill 04/23/2020
+ */
+
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+/* eslint-disable import/prefer-default-export */
+export const CovidFeature = () => (
+    <div className="feature-covid-container">
+        <h2 className="feature-covid__title">AN IMPORTANT MESSAGE ABOUT COVID-19</h2>
+        <div className="feature-covid">
+            <div className="feature-covid__body">
+                <div className="feature-covid__body-icon">
+                    <FontAwesomeIcon size="lg" icon="info-circle" />
+                </div>
+                <div className="feature-covid__body-content">
+                    <h1 className="feature-covid__body-title">
```

Comment:
This `<h1>` is more nested than the `<h2>` on line 12, and we've generally tried to avoid multiple h1s on a single page. I'd suggest making this another `h2` or even an `h3` (changing the `h2`s below it and so on, making sure it stays styled the same).